src/main/java/org/grpc/service/entities/Employee.java

```diff
@@ -0,0 +1,40 @@
+package org.grpc.service.entities;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString
+@EqualsAndHashCode
+public class Employee {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    private String firstName;
+    private String lastName;
+    @Column(unique = true)
```

Comment:
@Column(unique = true)  What is it for?

Reply:
It sets a parameter in the database's SQL so that no two working numbers can be the same - so everybody gets their own login. Column refers to the column in the DB